### PR TITLE
Refresh IG Group file POST addition & messaging improvements

### PR DESCRIPTION
**Pull request by echicoine-icf**
This pull request covers the following changes:

- Refresh IG POST expanded to include other resources such as group files. 
- Duplicate bundle files during refresh fixed.
- added -x as possible argument during Refresh IG to show expanded reporting for errors, warnings, and information messages.
- - Boolean "verboseMessaging" can be found throughout methods and when true, extra information is appended to output such as cql translation.
- Bundle Test Cases enhanced with multithreading optimizations.
- General enhancements to message output for readability.
- Renamed AbstractResourceProcessor to AbstractBundler. 
- - Decoupled MeasureProcessor from AbstractBundler, extends BaseProcessor
- - Created MeasureBundler, extends AbstractBundler
- - Renamed PlanDefinitionProcessor to PlanDefinitionBundler
- - Renamed QuestionnaireProcessor to QuestionnaireBundler
- Improved readability and complexity behind IGBundleProcessor instantiation. 
- Adjusted CqlTranslatorException to use List<CqlCompilerException>
- -Cql translation results adjusted for output such as:

image

- Github Issue:  
- [ ] I've read the contribution guidelines 
- [ ] Code compiles without errors
- [ ] Tests are created / updated
- [ ] Documentation is created / updated

By creating this PR you acknowledge that your contribution will be licensed under Apache 2.0

### CODE REVIEW

File: tooling/src/main/java/org/opencds/cqf/tooling/processor/IGTestProcessor.java [hunk 1]
```diff
@@ -172,105 +172,109 @@ public void testIg(TestIGParameters params) {
 
         CqfmSoftwareSystem testTargetSoftwareSystem =  getCqfRulerSoftwareSystem(params.fhirServerUri);
 
-        logger.info("Running IG test cases...");
+        System.out.println("\r\n[Running IG Test Cases]\r\n");
```
[the remark this answers is not in the document]
Comment (JPercival) [reply tooling/src/main/java/org/opencds/cqf/tooling/processor/IGTestProcessor.java:175]: The log format can be configured, and it's possible to simultaneously have different formats for different targets. For example, simple output that's redirected to the CLI output, and verbose output/formatting going to a log file. That is a separate level of configuration that's needed, but routing messaging through the logger API allows that type of configuration.

[the remark this answers is not in the document]
Comment (echicoine-icf) [reply tooling/src/main/java/org/opencds/cqf/tooling/processor/IGTestProcessor.java:175]: Sounds good to me! I'll go ahead and adjust this to use the logger and we can revisit cleaning up the format

**Comment by echicoine-icf** [comment]
Logger is now used over System.out

**Review by bryantaustin13 — APPROVED** [review]
LGTM

**Pull request merged**